lightly/api/api_workflow_compute_worker.py

```diff
@@ -489,3 +496,26 @@ def _snake_to_camel_case(snake: str) -> str:
     return components[0] + "".join(
         component.title() for component in components[1:]
     )
+
+
+def _validate_config(cfg: Optional[Dict[str, Any]], obj: Any) -> None:
+    """Validates that all keys in cfg are legit configuration options. """
```

Comment:
Please add more description about what this function is doing. It is quite non-trivial:
* I think it works only under certain assumptions on the type of `obj`
* It is recursive
* It suggests a best match